README.md

```diff
@@ -1,2 +1,29 @@
-# Hello-World
-my first repository test
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="testlogo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Hello World Project ReadMe</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+  - [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [👥 Author](#authos)
+
+
+# 📖 Hello World <a name="about-project"></a>
+
+**Hello World** is my first microverse project, which i start to understand the basics of GitHub.
+
+## 🛠 Built With HTML & CSS <a name="built-with"></a>
+
+### 👤 Author
+  
+**Nurbol Sultanov**
+
+- GitHub: [@everinurmind](https://github.com/everinurmind)
```

Comment:
- [ ] You did a good job implementing the `README` file :+1: but kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections are present in yours like key features, Future Features, Contributions, and  Getting Started that describe how a new developer could make use of your project by following some steps. It is important because it would make your project looks more professional and this is one of the first things that hiring managers or your potential colleagues will check when considering you for a position on their team. To learn more you could read [Document your repo in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/clean_repo.md)